app/uk/gov/hmrc/apidocumentation/config/ApplicationConfig.scala

```diff
@@ -16,59 +16,97 @@
-  lazy val ramlPreviewEnabled = runModeConfiguration.getBoolean(s"$env.features.ramlPreview").getOrElse(false)
-  lazy val ramlLoaderRewrites = buildRamlLoaderRewrites
-  lazy val showProductionAvailability = runModeConfiguration.getBoolean(s"$env.features.showProductionAvailability").getOrElse(false)
-  lazy val showSandboxAvailability = runModeConfiguration.getBoolean(s"$env.features.showSandboxAvailability").getOrElse(false)
-  lazy val productionApiHost = runModeConfiguration.getString("platform.production.api.host")
-  lazy val productionWwwHost = runModeConfiguration.getString("platform.production.www.host")
-  lazy val productionApiBaseUrl = platformBaseUrl("platform.production.api")
+  val developerFrontendBaseUrl = config.baseUrl("developer-frontend")
+  val thirdPartyDeveloperUrl = config.baseUrl("third-party-developer")
+  val securedCookie = config.getConfBool("cookie.secure", true)
+  val ramlPreviewEnabled = config.getConfBool("features.ramlPreview", false)
+  val ramlLoaderRewrites = buildRamlLoaderRewrites
+  val showProductionAvailability = config.getConfBool("features.showProductionAvailability", false)
+  val showSandboxAvailability = config.getConfBool("features.showSandboxAvailability", false)
+  val productionApiHost = config.getString("platform.production.api.host")
+  val productionWwwHost = config.getString("platform.production.www.host")
+  val productionApiBaseUrl = platformBaseUrl("platform.production.api")
 
-  lazy val sandboxApiHost = runModeConfiguration.getString("platform.sandbox.api.host")
-  lazy val sandboxWwwHost = runModeConfiguration.getString("platform.sandbox.www.host")
-  lazy val sandboxApiBaseUrl = platformBaseUrl("platform.sandbox.api")
-  lazy val sandboxWwwBaseUrl = platformBaseUrl("platform.sandbox.www")
+  val sandboxApiHost = config.getString("platform.sandbox.api.host")
+  val sandboxWwwHost = config.getString("platform.sandbox.www.host")
+  val sandboxApiBaseUrl = platformBaseUrl("platform.sandbox.api")
+  val sandboxWwwBaseUrl = platformBaseUrl("platform.sandbox.www")
 
-  lazy val title = "HMRC Developer Hub"
-  lazy val isStubMode = env == "Stub"
-  lazy val xmlApiBaseUrl = runModeConfiguration.getString(s"$env.xml-api.base-url").getOrElse("https://www.gov.uk")
+  val title = "HMRC Developer Hub"
+  val isStubMode = runMode.env == "Stub"
+  val xmlApiBaseUrl = config.getConfString("xml-api.base-url", "https://www.gov.uk")
 
-  lazy val apiDefinitionBaseUrl = baseUrl("api-definition")
+  val apiDefinitionBaseUrl = config.baseUrl("api-definition")
 
   private def buildRamlLoaderRewrites: Map[String, String] = {
-    Map(runModeConfiguration.getString(s"$env.ramlLoaderUrlRewrite.from").getOrElse("") ->
-      runModeConfiguration.getString(s"$env.ramlLoaderUrlRewrite.to").getOrElse(""))
+    Map(config.getConfString("ramlLoaderUrlRewrite.from", "") ->
+      config.getConfString("ramlLoaderUrlRewrite.to", ""))
   }
 
   private def platformBaseUrl(key: String) = {
```

Comment:
I think there may be a method in the standard HMRC lib that does this kind of thing

Reply:
I can't find one - what is it?

Reply:
Configuration in the Play library includes lots of deprecated methods for fetching config and giving sensible defaults.